challenges/2021_cloud_base_height/04_eval_pytorch.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>also very nice! I wonder what other ways there might be to present this, maybe bins levels in groups of ten and do a box and whiskers plot for each set of ten levels? theres quite a lot in this plot, which is very useful but hard to take it all in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
I split the text and plot eval up to help